ui/js/controllers/logviewer.js

```diff
@@ -158,17 +158,25 @@ logViewerApp.controller('LogviewerCtrl', [
                 if(artifactList.length > 0){
                     $scope.artifact = artifactList[0].blob;
 
-                    var revision = $scope.artifact.header.revision.substr(0,12);
-                    $scope.logRevisionFilterUrl = $scope.urlBasePath
-                        + "#/jobs?repo="
-                        + $scope.repoName + "&revision=" + revision;
-
-                    // Store the artifact epoch date string in a real date object for use
-                    var startTime = $scope.artifact.header.starttime;
-                    var startDate = new Date(0);
-                    startDate.setUTCSeconds(startTime);
-
-                    $scope.logDisplayDate = startDate.toString();
+                    $scope.logProperties = _.map(
+                        _.keys($scope.artifact.header).sort(), function(label) {
+                            var value = $scope.artifact.header[label];
+                            if (label === 'starttime') {
+                                // present start time as a human readable date
+                                var startDate = new Date(0);
+                                startDate.setUTCSeconds(value);
+                                value = startDate.toString();
+                            }
+                            return {label: label, value: value}
+                        });
+
+                    // linkify revision
+                    if ($scope.artifact.header.revision) {
+                        var revision = $scope.artifact.header.revision.substr(0,12);
+                        $scope.logRevisionFilterUrl = $scope.urlBasePath
+                            + "#/jobs?repo="
+                            + $scope.repoName + "&revision=" + revision;
```

Comment:
It's inherited from the existing code, but I wonder if [177-178] can be economized to a single line?

I checked the style of gluing via leading `+`'s but current linters don't seem to complain :) Our oft referenced [style guide](http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml#Code_formatting) seems to imply trailing `+`'s when wrapping, but I defer to you guys.